.buildkite/scripts/skip_if_only_docs_changes.sh

```diff
@@ -0,0 +1,17 @@
+# Source this file rather than fork-exec-ing it.
+
+only_docs_changes() {
+  test -n "$BUILDKITE_PULL_REQUEST_BASE_BRANCH" &&
+    git diff --quiet "refs/remotes/origin/$BUILDKITE_PULL_REQUEST_BASE_BRANCH.." -- \
```

Comment:
I would include a reference to [git's `pathspec` documentation](https://git-scm.com/docs/gitglossary#Documentation/gitglossary.txt-aiddefpathspecapathspec) since this is quite a new thing.

Reply:
ok